drivers/watchdog/Kconfig.nrfx

```diff
@@ -12,3 +12,17 @@ config WDT_NRFX
 	select NRFX_WDT
 	help
 	  Enable support for nrfx WDT driver for nRF MCU series.
+
+config NRFX_WDT0
+	bool "Enable WDT0 instance"
+	depends on HAS_HW_NRF_WDT || HAS_HW_NRF_WDT0
+	default y
+	 help
```

Comment:
Accidental extra space.

Maybe the help text could spell out "Watchdog timer" too.

Reply:
Compare with the `default` above.

Reply:
will be part of another PR